cloud/vsphere/templates.go

```diff
@@ -262,7 +262,6 @@ chmod a+rx /usr/bin/kubeadm.dl
 
 {{ define "configure" -}}
 KUBELET_VERSION={{ .Machine.Spec.Versions.Kubelet }}
-TOKEN={{ .Token }}
```

Comment:
(sanity check)
Have you confirmed that this is safe and runs? The code now generates the token but it still needs to get applied to the machine which (I thought) comes through this template.

Reply:
This is the masterStartupScript which no longer needs the token as it was only being used to pass into the machine controller. Only the nodes will need the token to join the cluster. There is a separate script for nodes (right above this one). I haven't had a chance to sanity test this yet as I'm focussing on GCP -- I was hoping that someone who has everything working can cherry pick (happy to sit next to them and verify it works, would just take me a really long time to get my environment setup).

Reply:
Finally found some time to try it. Your branch does work.